backend/src/main.py

```
@@ -81,11 +85,25 @@ def internal_error(error):
     logger.error(f"Internal server error: {error}")
     return error_response('Internal server error', 500)
 
+def run_async_services():
+    """Run asynchronous services in a dedicated event loop."""
+    loop = asyncio.new_event_loop()
+    asyncio.set_event_loop(loop)
+
+    # Start mempool monitoring
+    loop.create_task(mempool_monitor_service.start_monitoring())
```

Ruff (0.15.0)

[warning] Store a reference to the return value of `loop.create_task`

(RUF006)